lapce-ui/src/app.rs

```diff
@@ -247,159 +247,196 @@ fn macos_window_desc<T: druid::Data>(desc: WindowDesc<T>) -> WindowDesc<T> {
     };
 
     desc.menu(|_, _, _| {
-        Menu::new("Lapce")
-            .entry(
-                Menu::new("")
-                    .entry(MenuItem::new("About Lapce").command(Command::new(
-                        LAPCE_COMMAND,
-                        LapceCommand {
-                            kind: CommandKind::Workbench(
-                                LapceWorkbenchCommand::ShowAbout,
-                            ),
-                            data: None,
-                        },
-                        Target::Auto,
-                    )))
+        Menu::new("Lapce").entry(
+            Menu::new("")
+                .entry(MenuItem::new("About Lapce").command(Command::new(
+                    LAPCE_COMMAND,
+                    LapceCommand {
+                        kind: CommandKind::Workbench(
+                            LapceWorkbenchCommand::ShowAbout,
+                        ),
+                        data: None,
+                    },
+                    Target::Auto,
+                )))
+                .separator()
+                .entry(
+                    Menu::new("Settings...")
+                        .entry(MenuItem::new("Open Settings").command(Command::new(
+                            LAPCE_COMMAND,
+                            LapceCommand {
+                                kind: CommandKind::Workbench(
+                                    LapceWorkbenchCommand::OpenSettings,
+                                ),
+                                data: None,
+                            },
+                            Target::Auto,
+                        )).hotkey(SysMods::Cmd, ",")
+                        )
+                        .entry(MenuItem::new("Open Keyboard Shortcuts")
+                            .command(
+                            Command::new(
+                                LAPCE_COMMAND,
+                                LapceCommand {
+                                    kind: CommandKind::Workbench(
+                                        LapceWorkbenchCommand::OpenKeyboardShortcuts,
+                                    ),
+                                    data: None,
+                                },
+                                Target::Auto,
+                            ))
+                            .hotkey(SysMods::Cmd, "k")
```

Comment:
This deviates from our default keymap for macos
https://github.com/lapce/lapce/blob/8044853ff0d4dfd49b6e97815e4db8ca16b6f818/defaults/keymaps-macos.toml#L26-L28